src/pages/eth.tsx

```diff
@@ -212,7 +212,7 @@ const CardContainer = (props: FlexProps) => (
 )
 
 const TokenCard = (props: ComponentProps<typeof HorizontalCard>) => (
-  <HorizontalCard minW="full" my={2} mx={0} borderRadius={0} {...props} />
+  <HorizontalCard {...props} className="mx-0 my-2 min-w-full rounded-none" />
```

Comment:
We need to use [the `cn` function](https://github.com/ethereum/ethereum-org-website/blob/dev/src/lib/utils/cn.ts) here to merge the classNames. Otherwise, we will always override the className passed to `TokenCard`.
```suggestion
  <HorizontalCard className={cn("mx-0 my-2 min-w-full rounded-none", className)} {...props} />
```